src/playbackwindow.cpp

```diff
@@ -172,26 +177,34 @@ void playbackWindow::on_cmdSave_clicked()
 
 			qDebug("Free space: %llu  (%lu * %lu)", statvfsBuf.f_bsize * (uint64_t)statvfsBuf.f_bfree, statvfsBuf.f_bsize, statvfsBuf.f_bfree);
 			qDebug("Estimated file size: %llu", estimatedSize);
+			
 			qDebug("===================================");
 
 			statfs(camera->recorder->fileDirectory, &fileSystemInfoBuf);
 			bool fileOverMaxSize = (estimatedSize > 4294967296 && fileSystemInfoBuf.f_type == 0x4d44);//If file size is over 4GB and file system is FAT32
-			bool insufficientFreeSpace = (estimatedSize > (statvfsBuf.f_bsize * (uint64_t)statvfsBuf.f_bfree));
+			insufficientFreeSpace_estimate = (estimatedSize > (statvfsBuf.f_bsize * (uint64_t)statvfsBuf.f_bfree));
```

Comment:
This is a bit of a weird mixing of variable naming conventions. Combining CamelCase and snake_case all in one. We should probably just name this insufficientFreeSpaceEstimate for consistency.

Reply:
I thought this way of mixing them would be better because it would more clearly differentiate the two.  I could just change it to insufficientFreeSpaceEstimate, or change the other one to insufficientFreeSpace_current, because that one stores the current status of whether or not the amount of free space is insufficient. Would you still like me to change it to CamelCase?

Reply:
Please keep it all as CamelCase.
naming the pair of them as insufficientFreeSpaceCurrent and insufficientFreeSpaceEstimate sounds fine.